dist/stat_data.py

```diff
@@ -389,6 +394,32 @@ def __init__(self, name, desc, flags=''):
     LSMStat('lsm_work_units_done', 'tree maintenance operations executed'),
 
     ##########################################
+    # Performance Histogram Stats
+    ##########################################
+    PerfHistStat('perf_hist_readfs_latency_gt1000', 'file system read latency histogram - 1000ms+'),
+    PerfHistStat('perf_hist_readfs_latency_lt50', 'file system read latency histogram - 10-49ms'),
+    PerfHistStat('perf_hist_readfs_latency_lt100', 'file system read latency histogram - 50-99ms'),
+    PerfHistStat('perf_hist_readfs_latency_lt250', 'file system read latency histogram - 100-249ms'),
+    PerfHistStat('perf_hist_readfs_latency_lt500', 'file system read latency histogram - 250-499ms'),
+    PerfHistStat('perf_hist_readfs_latency_lt1000', 'file system read latency histogram - 500-999ms'),
+    PerfHistStat('perf_hist_readop_latency_gt10000', 'operation read latency histogram - 10000us+'),
+    PerfHistStat('perf_hist_readop_latency_lt250', 'operation read latency histogram - 100-249us'),
+    PerfHistStat('perf_hist_readop_latency_lt500', 'operation read latency histogram - 250-499us'),
+    PerfHistStat('perf_hist_readop_latency_lt1000', 'operation read latency histogram - 500-999us'),
+    PerfHistStat('perf_hist_readop_latency_lt10000', 'operation read latency histogram - 1000-9999us'),
+    PerfHistStat('perf_hist_writefs_latency_gt1000', 'file system write latency histogram - 1000ms+'),
+    PerfHistStat('perf_hist_writefs_latency_lt50', 'file system write latency histogram - 10-49ms'),
+    PerfHistStat('perf_hist_writefs_latency_lt100', 'file system write latency histogram - 50-99ms'),
+    PerfHistStat('perf_hist_writefs_latency_lt250', 'file system write latency histogram - 100-249ms'),
+    PerfHistStat('perf_hist_writefs_latency_lt500', 'file system write latency histogram - 250-499ms'),
+    PerfHistStat('perf_hist_writefs_latency_lt1000', 'file system write latency histogram - 500-999ms'),
+    PerfHistStat('perf_hist_writeop_latency_gt10000', 'operation write latency histogram - 10000us+'),
+    PerfHistStat('perf_hist_writeop_latency_lt250', 'operation write latency histogram - 100-249us'),
+    PerfHistStat('perf_hist_writeop_latency_lt500', 'operation write latency histogram - 250-499us'),
+    PerfHistStat('perf_hist_writeop_latency_lt1000', 'operation write latency histogram - 500-999us'),
+    PerfHistStat('perf_hist_writeop_latency_lt10000', 'operation write latency histogram - 1000-9999us'),
+
```

Comment:
I have a preference for the variables to be called `perf_hist_fsread*` and `perf_hist_fswrite*` and `perf_hist_opread*` and `perf_hist_opwrite*` only so that the ones that go together are grouped together.